pynumdiff/tests/test_diff_methods.py

```diff
@@ -150,7 +150,7 @@ def spline_irreg_step(*args, **kwargs): return splinediff(*args, **kwargs)
                         [(0, 0), (1, 1), (0, 0), (1, 1)],
                         [(1, 0), (2, 2), (1, 0), (2, 2)],
                         [(1, 0), (3, 3), (1, 0), (3, 3)]],
-    spectraldiff: [[(-15, -15), (-14, -15), (0, -1), (0, 0)],
+    spectraldiff: [[(-15, -15), (-14, -14), (0, -1), (0, 0)],
```

Comment:
`spectraldiff` uses convolution, which is now based on scipy's `convolve1d` rather than numpy's `convolve`, which changes its answers ever so slightly.